encore.cabal

```diff
@@ -48,5 +48,5 @@ executable encorec
                , boxes
                , filepath
   hs-source-dirs:      src/back src/front src/ir src/opt src/parser src/types
-  ghc-options: -Werror
+  ghc-options: -Werror -fmax-pmcheck-iterations=10000000
```

Comment:
Can I ask what this is for? Not against it, just curious!

Reply:
Encountered some warning without it; more at https://ghc.haskell.org/trac/ghc/ticket/11822

Reply:
Between GHC 7.10 and 8.0.2 they lowered the limit, we raises the limit back up to handle the warning we otherwise get from putChildren.

.../encore/src/ir/AST/Util.hs:145:1: warning:
   Pattern match checker exceeded (2000000) iterations in
   an equation for ‘putChildren’. (Use -fmax-pmcheck-iterations=n
   to set the maximun number of iterations to n)